class/person.rb

```diff
@@ -1,10 +1,14 @@
 require 'securerandom'
+require '../nameable'
+require '../trimmer_decorator'
+require '../capitalize_decorator'
 
-class Person
```

Comment:
User is unable to correct the name if the characters are more than 10 and also capitalization does not work. Kindly see the screenshot of errors below.

Reply:
Ohhh yeah, I broke the code solving a linter warning with the `super` call. Solved 👍🏽